AirCasting/Dashboard/ReorderingDashboardViewModel.swift

```diff
@@ -0,0 +1,24 @@
+// Created by Lunar on 28/01/2022.
+//
+
+import Foundation
+
+class ReorderingDashboardViewModel: ObservableObject {
+    @Published var sessions: [SessionEntity]
+    
+    @Published var currentSession: SessionEntity?
```

Comment:
but let's rename this - `currentSession` is not the greatest name, maybe add a note of domain here and make it smth like `currentlyDraggedSession`?